code/game/machinery/modular_shield.dm

```diff
@@ -585,12 +601,28 @@
 		connected.shield_generator = null
 		connected.update_icon_state()
 
+/obj/machinery/modular_shield/module/node/cable
+
```

Comment:
why is there a newline here

Reply:
fatigue

Reply:
apparently thats how i wrote all of them